src/widgets/modals/CashTransactionModal.js

```diff
@@ -42,7 +42,7 @@ export const CashTransactionModal = ({ onConfirm }) => {
   const [isDescriptionModalOpen, setIsDescriptionModalOpen] = useState(false);
 
   const names = useMemo(() => UIDatabase.objects('CashTransactionName'));
-  const reasons = useMemo(() => UIDatabase.objects('Options'));
+  const reasons = useMemo(() => UIDatabase.objects('CashTransactionReason'));
```

Comment:
this needs a dependency list, and the one above - `[]`

Reply:
Good catch, de740a0e079c65cdf001008e3d98583422ff2a7e.

Reply:
https://github.com/openmsupply/mobile/blob/2ca29f7680485d3922950388f263c7b7b78c0477/src/widgets/modals/CashTransactionModal.js#L63-L64 

https://github.com/openmsupply/mobile/blob/2ca29f7680485d3922950388f263c7b7b78c0477/src/widgets/modals/CashTransactionModal.js#L66-L83

Maybe more that I didn't see !